recipes/ncurses/ncurses.inc

```diff
@@ -37,6 +37,8 @@ EXTRA_OECONF += "\
 --without-cxx-binding \
 --with-terminfo-dirs=${sysconfdir}/terminfo:${datadir}/terminfo \
 --enable-overwrite \
+--enable-pc-files \
+--with-pkg-config-libdir=${libdir}/pkgconfig \
```

Comment:
Correct me if I'm wrong, but it seems pkgconfig.oeclass is for setting up appropriate env variables and ensuring a pkg-config binary gets staged. This patch is not about using .pc files from the stuff ncurses depends on; it is about convincing ncurses' build system to provide .pc files for others (those that depend on ncurses) to use.

```
  --with-pkg-config-libdir=XXX use given directory for installing pc-files
  --enable-pc-files       generate and install .pc files for pkg-config
```